### DISCUSSION TITLE
Can I use rowy with Notion?

### DISCUSSION THREAD

**Discussion by KKimj** [AntlerVC/firetable#726] [Support / Q&A] [answered]
Hello!!
I love this project, rowy!

Can I use rowy with Notion, sync with firebase?
Something like below?

https://zapier.com/apps/firebase/integrations/notion

Thanks!!
Have a Nice a day!

**Comment by harinij** [comment, marked as answer]
@KKimj glad to hear you are enjoying Rowy :)

Check this [docs](https://docs.rowy.io/extensions) on how you can listen to any Firestore data changes such as create/update/delete easily at field level and flexibly call any API like notion to send data, create an item etc. We have a few example for sendgrid, discord [here](https://demo.rowy.io/table/userManagement#tableModal=%22extensions%22).

**Reply by KKimj** [reply]
Wow ! 
Thanks your insight!
